### PR TITLE
whisper-fill.py: python3 compatiblity with itertools

### PR DESCRIPTION
**Pull request by piotr1212**
Should address issues mentioned in https://github.com/graphite-project/whisper/issues/190

### CODE REVIEW

File: bin/whisper-fill.py [hunk 1]
```diff
@@ -23,13 +23,14 @@
...
 
-import itertools
 import time
 import sys
 import optparse
 
 if sys.version_info >= (3, 0):
     xrange = range
+else:
+    from future_builtins import filter
```
Comment (deniszh) [inline bin/whisper-fill.py]: Should include zip too?

Comment (piotr1212) [inline bin/whisper-fill.py]: Not 100% sure if it is needed in this specific case. But I've added it to be sure we get same behaviour in all version.

**Comment by piotr1212** [comment]
I have no clue why the travis test fails, seems to work fine locally.

**Comment by deniszh** [comment]
Test went fine after restart, Codacy also can be ignored in this case

**Pull request merged**